niworkflows/viz/utils.py

```diff
@@ -215,7 +215,7 @@ def _3d_in_file(in_file):
     try:
         in_file = nb.load(in_file)
     except AttributeError:
-        in_file = in_file
+        pass
```

Comment:
I have no idea how an `AttributeError` would arise here. I think we should just drop the try/except.

Reply:
It was added in https://github.com/nipreps/niworkflows/commit/9ba1112845142b847b197f7d40a3a8193a347783 without any explanation.

Of the few places that raise an AttributeError in nibabel:

[**nibabel/nibabel/openers.py**](https://github.com/nipy/nibabel/blob/0e03b9403216af238296d5aa7e44f8f64242f896/nibabel/openers.py#L231-L235)
Lines 231 to 235 of [0e03b94](https://github.com/nipy/nibabel/blob/0e03b9403216af238296d5aa7e44f8f64242f896/)
```python
    def readinto(self, buffer: WriteableBuffer, /) -> int | None:
        # Check and raise our own error for type narrowing purposes
        if hasattr(self.fobj, 'readinto'):
            return self.fobj.readinto(buffer)
        raise AttributeError(f'{self.fobj.__class__.__name__} has no attribute "readinto"')
```

Reply:
Removed the `AttributeError` check.